internal/api/router.go

```diff
@@ -86,6 +86,12 @@ func (r *Router) Routes(rg *echo.Group) {
 		v2.PATCH("/roles/:role_id", r.roleV2Update)
 		v2.DELETE("/roles/:id", r.roleV2Delete)
 
+		v2.GET("/resources/:id/role-bindings", r.roleBindingsList)
+		v2.GET("/resources/:id/role-bindings/:rb_id", r.roleBindingGet)
```

Comment:
The resource parameter is redundant here, IMO.

```suggestion
		v2.GET("/role-bindings/:rb_id", r.roleBindingGet)
```